src/main/java/twilightforest/entity/EntityTFTowerGhast.java

```diff
@@ -264,15 +264,15 @@ protected void updateAITasks() {
 	}
 
 	public int getAttackStatus() {
-		return dataManager.get(ATTACK_STATUS);
+		return (int)dataManager.get(ATTACK_STATUS);
 	}
 
 	public int getAttackTimer() {
-		return dataManager.get(ATTACK_TIMER);
+		return (int)dataManager.get(ATTACK_TIMER);
 	}
 
 	public int getPrevAttackTimer() {
-		return dataManager.get(ATTACK_PREVTIMER);
+		return (int)dataManager.get(ATTACK_PREVTIMER);
 	}
```

Comment:
these are unnecessary

Reply:
Eclipse reported a compile error. Maybe it's an IDE version dependent issue. I use Eclipse Mars.

Feel free to remove these casts when merging.